particles/Restaurants/source/RestaurantFind.js

```diff
@@ -53,7 +53,7 @@ defineParticle(({UiParticle}) => {
         name: p.name,
         icon: p.icon,
         address: p.vicinity,
-        rating: p.rating,
+        rating: Math.round(p.rating*100)/100,
```

Comment:
haha, i was demo-ing it today and it looked funny 😃

Reply:
Lol, my first thought was "OMG ui-broker is clobbering the data!" but then I looked at the packets coming from Google and the crazy ratings were sitting there.